src/main/java/io/github/mzmine/datamodel/features/types/graphicalnodes/FeatureShapeMobilogramChart.java

```diff
@@ -107,16 +103,7 @@ public FeatureShapeMobilogramChart(@NotNull ModularFeatureListRow row, AtomicDou
     chart.getXYPlot().getDomainAxis().setRange(defaultRange);
     chart.getXYPlot().getDomainAxis().setDefaultAutoRange(defaultRange);
 
-    BufferedImage img = chart.getChart()
-        .createBufferedImage(GraphicalColumType.DEFAULT_GRAPHICAL_CELL_WIDTH,
-            GraphicalColumType.DEFAULT_GRAPHICAL_CELL_HEIGHT);
-    ImageView view = new ImageView(SwingFXUtils.toFXImage(img, null));
-    view.setOnMouseClicked(e -> MZmineCore.runLater(() -> {
-      getChildren().remove(view);
-      getChildren().add(chart);
-      e.consume();
-    }));
-
-    MZmineCore.runLater(() -> getChildren().add(view));
+    setChartCreateImage(chart, GraphicalColumType.DEFAULT_GRAPHICAL_CELL_WIDTH,
+        GraphicalColumType.DEFAULT_IMAGE_CELL_HEIGHT);
```

Comment:
GraphicalColumType.DEFAULT_GRAPHICAL_CELL_HEIGHT, mobilograms dont need to be as high

Reply:
True copy paste error

Reply:
changed back to DEFAULT_GRAPHICAL_CELL_HEIGHT